socorro/processor/mozilla_transform_rules.py

```diff
@@ -697,34 +705,52 @@ def _get_version_data(self, product, build_id, channel):
 
         return None
 
+    def is_final_beta(self, version):
+        """Denotes whether this version could be a final beta"""
+        # NOTE(willkg): this started with 26.0 and 38.0.5 was an out-of-cycle
+        # exception
+        return version > 26.0 and (version.endswith('.0') or version == '38.0.5')
```

Comment:
This logic was effectively copied from ftpscraper. The assumption is that it was right there, so it's probably right here, too.